tests/e2e/performance/mcg/test_mcg_cosbench.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, performance
+from ocs_ci.ocs.cosbench import Cosbench
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def cosbench(request):
+
+    cosbench = Cosbench()
+
+    def teardown():
+        cosbench.cosbench_teardown()
+
+    request.addfinalizer(teardown)
+    return cosbench
+
+
+@performance
+@pytest.mark.polarion_id("OCS-3694")
+class TestMCGCosbench(E2ETest):
+    """
+    Test cosbench workloads for MCG
+    """
+
+    def test_mcg_cosbench_performance(self, cosbench):
+        """
+        This test to perform reads and write objects to a bucket with multiple of samples and sizes.
+        The operation will be defined with number of % read and write.
+        After running main workload, performance numbers will be collected and saved to a spreadsheet for
+        performance analysing.
+        """
+
+        bucket_prefix = "bucket-"
+        buckets = 1
+        objects = 100
```

Comment:
IMO, from performance point of view, 100 objects is not enough.

Reply:
Increase objects number to objects = 10000